### PR TITLE
Mod 9338 add iija spending to award search table view

### PR DESCRIPTION
**Pull request by chasls2**
**High level description:**

Updated the Recipient Search:
Added infrastructure Layout and Obligation. 

**Technical details:**

I added mock data that can be seen while testing instead of ResultsRableContainer Line 161-164 can be commented out and replaced with const parsedResults = mockData.results;

**JIRA Ticket:**
[DEV-9338](https://federal-spending-transparency.atlassian.net/browse/DEV-9338)

**Mockup:**
https://invis.io/RYA3XN5WP#/273832670_Homepage_2-2_E

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [x] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/js/containers/search/table/mockData.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const mockData = {
```
Comment (anjenkin) [inline src/js/containers/search/table/mockData.js:1]: The mock data can be deleted.  Though you'll need to screenshare to demo this working for QA

**Review by anjenkin — CHANGES_REQUESTED** [review]
See inline...

File: src/js/containers/search/table/ResultsTableContainer.jsx [hunk 2]
```diff
@@ -361,7 +361,8 @@ export class ResultsTableContainer extends React.Component {
                     ...result,
                     generated_internal_id: encodeURIComponent(result.generated_internal_id)
                 }));
-
+                // can test mock datat with this
```
Comment (anjenkin) [inline src/js/containers/search/table/ResultsTableContainer.jsx:364]: references to mock data can be removed

File: src/js/containers/search/table/mockData.js (hunk 1 above)
Comment (chasls2) [reply src/js/containers/search/table/mockData.js:1]: will do

File: src/js/containers/search/table/ResultsTableContainer.jsx (hunk 2 above)
Comment (chasls2) [reply src/js/containers/search/table/ResultsTableContainer.jsx:364]: will do after demo

Comment (anjenkin) [reply src/js/containers/search/table/ResultsTableContainer.jsx:364]: don't forget this one too

**Review by anjenkin — CHANGES_REQUESTED** [review]
I just noticed that when sorting the columns there's an error.  Originally I thought this was due to the lack of API support but the fields are in the API so we'll have to figure out the issue with sorting.

**Review by anjenkin — APPROVED** [review]

**Pull request merged**